MyDataHelpsKit/Model/SurveyAnswers.swift

```diff
@@ -94,8 +102,10 @@ public struct SurveyAnswer: Decodable {
     public let surveyName: String
     /// Name of the survey displayed to the participant.
     public let surveyDisplayName: String
-    /// Date and time at which the survey answer was submitted. Different from the time at which the whole survey was submitted.
+    /// Date and time at which the survey answer was recorded by the participant.
     public let date: Date?
+    /// Date and time at which the entire survey was submitted.
+    public let insertedDate: Date
```

Comment:
Our documentation here is accurate. However, in troubleshooting an issue recently, I was comparing date timestamps from the answer as well as insertedDates and encountered a situation where they were a little out of order (< 60 second). My realization was this is because `insertedDate` uses the server clock whereas `date` and other answer time stamps use whatever date is passed from the client. It is possible that the clocks are not in sync. Perhaps the nuance is apparent as "recorded by the participant" precludes this date is coming from the client?

Reply:
Good point. Pushed a commit with various documentation updates cribbed from https://github.com/CareEvolution/rkstudioapi-docs/pull/140, including a paragraph that explains the difference in SurveyAnswer dates.